### DISCUSSION TITLE
Default User/Password?

### DISCUSSION THREAD

**Discussion by WCRalph** [TomBursch/kitchenowl#278] [Q&A]
I've just installed KitchenOwl through docker, and the login page loads up fine, but it just says "Username", "Password" and a Login button.  Nothing else.

Is there a default username/password somewhere?  I've been searching but not finding anything.

**Comment by TomBursch** [comment]
You should see an onboarding page when no user is present on the server. Does your backend run into any errors?

**Reply by WCRalph** [reply]
So I thought everything was working since the page showed up, but you're right the database was causing errors for being "locked".  I was having the data folder mounted from a SMB share and the database.db file was 0.  Tried it to a local directory and it worked and built the db.

Then I tried copying the db to the SMB share, KitchenOwl started up with "Let's create a user" page, but it won't complete because of "locked" db again.

So I have to figure out what's going on there.

Thanks.
